tests/acceptance-test/src/test/java/admin/cmd/StartupSteps.java

```diff
@@ -20,53 +20,58 @@ public StartupSteps() {
-            params.put("unixSocketPath", ipcFile.toAbsolutePath().toString());
-            Path configFile = ElUtil.createTempFileFromTemplate(url, params);
-            Party party = new Party("", configFile.toUri().toURL(), "X");
-            Config config = party.getConfig();
-            assertThat(emptyKeyFile).exists();
+        Given(
+                "configuration file with with key paths containing empty values",
+                () -> {
+                    URL url = getClass().getResource("/empty-keyspath-config.json");
+                    Path emptyKeyFile = Files.createTempFile(UUID.randomUUID().toString(), "");
+                    Path ipcFile = Files.createTempFile(UUID.randomUUID().toString(), "");
+                    Map<String, String> params = new HashMap<>();
+                    params.put("emptyKeyFilePath", emptyKeyFile.toAbsolutePath().toString());
+                    params.put("unixSocketPath", ipcFile.toAbsolutePath().toString());
+                    Path configFile = ElUtil.createTempFileFromTemplate(url, params);
+                    Party party = new Party("", configFile.toUri().toURL(), "X");
+                    Config config = party.getConfig();
+                    assertThat(emptyKeyFile).exists();
 
-            partyHolder.add(party);
-        });
+                    partyHolder.add(party);
+                });
 
         List<ExecutionResult> results = new ArrayList<>();
-        When("admin user executes start", () -> {
-            assertThat(partyHolder).hasSize(1);
-            Party party = partyHolder.iterator().next();
-            ExecutionResult result = Utils.start(party);
-            results.add(result);
-        });
+        When(
+                "admin user executes start",
+                () -> {
+                    assertThat(partyHolder).hasSize(1);
+                    Party party = partyHolder.iterator().next();
+                    ExecutionResult result = Utils.start(party);
+                    results.add(result);
+                });
 
-        Then("node returns error message and exits", () -> {
-            assertThat(results).hasSize(1);
-            ExecutionResult result = results.get(0);
-            assertThat(result.getExitCode()).isNotEqualTo(0);
-            assertThat(result.getOutput()).hasSize(2);
-            
-            assertThat(result.getOutput())
-                    .anyMatch(m -> m.startsWith("Config validation issue: keys.keyData[0].privateKey"));
-            
-            assertThat(result.getOutput())
-                    .anyMatch(m -> m.startsWith("Config validation issue: keys.keyData[0].publicKey"));
+        Then(
+                "node returns error message and exits",
+                () -> {
+                    assertThat(results).hasSize(1);
+                    ExecutionResult result = results.get(0);
+                    assertThat(result.getExitCode()).isNotEqualTo(0);
+                    assertThat(result.getOutput()).hasSize(3);
```

Comment:
If a new error is going to be output, we should add it to the list below to make sure it's the error we expect.